src/test/resources/changesets/35_delete_client_scope_mappers.yml

```diff
@@ -0,0 +1,15 @@
+id: delete-client-scope-mappers
+author: janunld
+realm: integ-test
+changes:
+  - addClientScope:
+      name: testMappers
+  - addClientScopeAudienceMapper:
```

Comment:
if we don't delete the previously created clientScope this command fails because a mapper with this name already exists in the clientScope.